lrauv_ignition_plugins/proto/lrauv_state.proto

```diff
@@ -33,66 +33,114 @@ import "ignition/msgs/header.proto";
 
+  /// \brief Not populated
   int32 errorPad_                 = 2;
+
+  /// \brief Not populated
   int32 utmZone_                  = 3;
+
+  /// \brief Not populated
   int32 northernHemi_             = 4;
-  float propOmega_                = 5;  // Current angular velocity of the
-                                        // propeller. Unit: rad / s. Positive
-                                        // values rotate the propeller
-                                        // clockwise when looking from the back,
-                                        // and propel the vehicle forward.
-  float propThrust_               = 6;  // Not populated
-  float propTorque_               = 7;  // Not populated
-  float rudderAngle_              = 8;  // Angle that the rudder joint is
-                                        // currently at. Unit: radians.
-                                        // Higher values have the vertical fins
-                                        // rotated more clockwise when looking
-                                        // from the top (i.e. to starboard)
+
+  /// \brief Current angular velocity of the propeller. Unit: rad / s. Positive
+  /// values rotate the propeller clockwise when looking from the back,
+  /// and propel the vehicle forward.
+  float propOmega_                = 5;
+
+  /// \brief Not populated
+  float propThrust_               = 6;
+
+  /// \brief Not populated
+  float propTorque_               = 7;
+
+  /// \brief Angle that the rudder joint is currently at. Unit: radians.
+  /// Higher values have the vertical fins rotated more clockwise when looking
+  /// from the top (i.e. to starboard)
+  float rudderAngle_              = 8;
+
+  /// \brief Angle that the elevator joint is currently at. Unit: radians.
+  /// Higher values have the horizontal fins rotated more counter-clockwise when
+  /// looking from starboard, which tilts the nose downward when moving forward.
   float elevatorAngle_            = 9;
-  float massPosition_             = 10;  // Position of the battery's joint.
-                                         // Unit: meters. Positive values have
-                                         // the battery forward, tilting the
-                                         // nose downward.
-  float buoyancyPosition_         = 11;  // Volume of the VBS. Unit: cubic
-                                         // meters. Volumes higher than the
-                                         // neutral volume push the vehicle
-                                         // upwards.
+
+  /// \brief Position of the battery's joint. Unit: meters. Positive values have
+  /// the battery forward, tilting the nose downward.
+  float massPosition_             = 10;
+
+  /// \brief Volume of the VBS. Unit: cubic meters. Volumes higher than the
+  /// neutral volume push the vehicle upwards.
+  float buoyancyPosition_         = 11;
+
+  /// \brief Vertical position of the vehicle with respect to sea level. Higher
```

Comment:
```suggestion
  /// \brief Vertical position of the vehicle with respect to mean sea level. Higher
```

Sea level is ambiguous with tides. There's MSL or there's depth to current surface, or several other semi-obscure options. This could be alternatively switched to a TODO.